client/actions/projects.js

```diff
@@ -106,21 +106,54 @@ export function updateAndSave(data) {
   };
 }
 
+const doConditionsUpdate = (data, dispatch, state) => {
+  const { application: { basename } } = state;
+  const params = {
+    method: 'PUT',
+    url: `${basename.replace(/\/edit?/, '')}/conditions`,
+    data
+  };
+  return Promise.resolve()
+    .then(() => sendMessage(params))
+    .catch(err => {
+      console.error(err);
```

Comment:
Do we care about this? I suspect it might actually be useful so I'm happy to leave it.